app/src/main/res/layout/activity_main.xml

```diff
@@ -41,6 +41,7 @@
         android:layout_height="wrap_content"
         android:layout_gravity="bottom|end"
         android:layout_margin="@dimen/fab_margin"
+        android:onClick="onClickFab"
```

Comment:
FAB = Floating Action Button right? not sure if it's the most understandable name :p